fli-iam/shanoir-ng
# Shanoir issue#1466 : derived studies

Allow the copy of dataset from one study (source) to another (target)

**How to test**

Prerequisites:
- You must be part of both studies (source and target) to copy datasets
- You must have admin rights on both studies
- You can't copy datasets that are copied from another study
- You can't copy a dataset to the study the dataset comes from
- You can't copy a dataset to a study that already contains the dataset
- You can't copy a processed dataset

Tests:
Login on shanoir with a user account
Join a study (source) and get admin rights
Create a new study (target)
Go to solr and select the dataset you want to copy
Click on the "Copy to study" button in the command bar above the table
Select the study (target) you want to copy those datasets to
A status message "Copy worked" is displayed once the copy is done

Check that everything worked:
Every center and subject from the copied dataset are added to the new study
In study details, the 'tree' tab is filled with the examinations, acquisitions and datasets from the copied datasets
